pkg/services/sqlstore/migrations/ualert/route_matcher.go

```diff
@@ -0,0 +1,195 @@
+package ualert
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+	"github.com/prometheus/alertmanager/pkg/labels"
+	"xorm.io/xorm"
+)
+
+type migrateRoutes struct {
+	migrator.MigrationBase
+	mg *migrator.Migrator
+}
+
+func (m *migrateRoutes) SQL(dialect migrator.Dialect) string {
+	return "code migration"
+}
+
+func (m *migrateRoutes) Exec(sess *xorm.Session, mg *migrator.Migrator) error {
+	configs := []AlertConfiguration{}
+	err := sess.SQL(`SELECT * FROM alert_configuration`).Find(&configs)
```

Comment:
Kind of hard to say if should be modifying the latest config per org, or all the configs:
 - In context of "backup", only recent makes sense
 - In context of UI building on this feature, they should all change, so all configs could be read as the same object type/schema if/when such a feature is created.